src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -186,36 +180,36 @@ def test_forecast_run_cmd():
     """
     Tests that the command to be used to run the forecast executable was built successfully.
     """
-    hera_expected = "srun --export=ALL test_exec.py"
-    assert hera_expected == FV3Forecast().run_cmd(
-        "--export=ALL", run_cmd="srun", exec_name="test_exec.py"
-    )
-
-    cheyenne_expected = "mpirun -np 4 test_exec.py"
-    assert cheyenne_expected == FV3Forecast().run_cmd(
-        "-np", 4, run_cmd="mpirun", exec_name="test_exec.py"
-    )
-
-    wcoss2_expected = "mpiexec -n 4 -ppn 8 --cpu-bind core -depth 2 test_exec.py"
-    assert wcoss2_expected == FV3Forecast().run_cmd(
-        "-n",
-        4,
-        "-ppn",
-        8,
-        "--cpu-bind",
-        "core",
-        "-depth",
-        2,
-        run_cmd="mpiexec",
-        exec_name="test_exec.py",
-    )
+    with patch.object(FV3Forecast, "_validate", return_value=True):
+        fcstobj = FV3Forecast(config_file="/not/used")
+        hera_expected = "srun --export=ALL test_exec.py"
+        assert hera_expected == fcstobj.run_cmd(
+            "--export=ALL", run_cmd="srun", exec_name="test_exec.py"
+        )
+        cheyenne_expected = "mpirun -np 4 test_exec.py"
+        assert cheyenne_expected == fcstobj.run_cmd(
+            "-np", 4, run_cmd="mpirun", exec_name="test_exec.py"
+        )
+        wcoss2_expected = "mpiexec -n 4 -ppn 8 --cpu-bind core -depth 2 test_exec.py"
+        assert wcoss2_expected == fcstobj.run_cmd(
+            "-n",
+            4,
+            "-ppn",
+            8,
+            "--cpu-bind",
+            "core",
+            "-depth",
+            2,
+            run_cmd="mpiexec",
+            exec_name="test_exec.py",
+        )
 
 
 @pytest.mark.parametrize("section", ["static", "cycledep"])
 @pytest.mark.parametrize("link_files", [True, False])
-def test_stage_files(tmp_path, section, link_files):
+def test__stage_files(tmp_path, section, link_files):
```

Comment:
It doesn't seem that `stage_files()` became private after all.

Reply:
Fixing.